### PR TITLE
Nutrition number tweaks

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Reduced the slowdown on fat, and metabolism rates related to satiation.
Being fat increases stamina damage and reduces metabolism rate (increases the time chemicals stay in your body)
Being hungry doubles your metabolism speed.
Values on all hunger related moodlets halved.

## Why It's Good For The Game

First of all, fat slowdown seems ridiculous. 50%? Really?

Previous metabolism made no sense. It would slow down when you were hungry (meaning healing chems would stay longer) and increase when you were fat. This isn't very realistic either; being fat/overfed slows down the rate your body handles chemicals, while being hungry increases it.

Gameplay wise makes no sense either. Being hungry increases the efficiency of all healing chems by 20% (they stay longer in your body), being well fed actually has the opposite effect and makes healing chems last 20% shorter... And being fat incurs the dreaded 33% slowdown that everyone hates. And you have the moodlets slowing you down on top of that.

And on top of that it's really easy to get fat, too. Just eat a burger that has been cooked with a T4 microwave that has 100u nutriment that, by the time they synthesize, you will have 3 times the amount of body fat.

## Changelog
:cl:
tweak: halved the mood effect of positive and negative hunger related moodlets
tweak: reduced fat slowdown
add: being hungry doubles all incoming stamina damage
tweak: being hungry doubles metabolism rate
tweak: being fat reduces metabolism rate
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1117,21 +1117,26 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 		if(H.overeatduration > 1)
 			H.overeatduration -= 2 //doubled the unfat rate
 
+	var/metabomism_fat = 0.75
+	var/metabomism_regular = 1
+	var/metabomism_vigor = 1
+	var/metabomism_hungry = 2
+
```
Comment (PowerfulBacon) [inline code/modules/mob/living/carbon/human/species.dm:1124]: These should be defines

File: code/modules/mob/living/carbon/human/species.dm [hunk 2]
```diff
@@ -1535,6 +1540,8 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	var/Iforce = I.force //to avoid runtimes on the forcesay checks at the bottom. Some items might delete themselves if you drop them. (stunning yourself, ninja swords)
 
 	var/weakness = H.check_weakness(I, user)
+	if (I.damtype == STAMINA && H.nutrition < NUTRITION_LEVEL_STARVING)
+		I.force*=2 //stamina fat multiplier under NUTRITION_LEVEL_STARVING
```
Comment (PowerfulBacon) [inline code/modules/mob/living/carbon/human/species.dm:1544]: Double stamina damage is pretty crazy, I would go for values like 1.2 for starving and 1.4 for super starving or whatever.
Also why would being hungry make you take more stamina damage?

**Comment by utilbird** [comment]
Why would being hungry _increase_ your metabolism

File: code/modules/mob/living/carbon/human/species.dm (hunk 2 above)
Comment (eeSPee) [reply code/modules/mob/living/carbon/human/species.dm:1544]: Well I thought of it as an incentive for people to eat, people wouldn't stay hungry for the buff, and muscles have less protein and energy so they tire faster from stamina.

**Comment by eeSPee** [comment]
maybe not a good idea...

**Pull request closed without merging**